features/admin.applications.v1/components/shared-access-status-banner.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Alert from "@oxygen-ui/react/Alert";
+import AlertTitle from "@oxygen-ui/react/AlertTitle";
+import Box from "@oxygen-ui/react/Box";
+import Button from "@oxygen-ui/react/Button";
+import { ModalWithSidePanel } from "@wso2is/admin.core.v1/components/modals/modal-with-side-panel";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Heading, LinkButton } from "@wso2is/react-components";
+import React, { ReactElement, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { Grid } from "semantic-ui-react";
+import ApplicationShareStatusWizard from "./wizard/application-share-status-wizard";
+import { OperationStatus } from "../constants/application-management";
+import { OperationStatusSummary } from "../models/application";
+
+/**
+ * Proptypes for the operation status banner component.
+ */
+interface OperationStatusBannerPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Application share operation status.
+     */
+    status: OperationStatus;
+    /**
+     * The operation ID of the share operation.
+     */
+    sharingOperationId?: string;
+    /**
+     * The summary of the share operation.
+     */
+    sharingOperationSummary?: OperationStatusSummary;
+}
+
+/**
+ * Operation status banner component.
+ *
+ * @param props - Props injected to the component.
+ *
+ * @returns Operation status banner component.
+ */
+export const OperationStatusBanner: React.FC<OperationStatusBannerPropsInterface> = (
+    props: OperationStatusBannerPropsInterface
+): ReactElement => {
+    const { status, sharingOperationId, [ "data-componentid" ]: componentId, sharingOperationSummary } = props;
+    const [ showStatusModal, setShowStatusModal ] = useState(false);
+    const { t } = useTranslation();
+
+    const handleShowStatusModal = () => {
+        setShowStatusModal(true);
+    };
+    const handleHideStatusModal = () => {
+        setShowStatusModal(false);
+    };
+
+    return (
+
+        <div className="banner-wrapper">
+            { status === OperationStatus.IN_PROGRESS && (
+                <div className="banner-wrapper">
+                    <Alert
+                        severity="warning"
+                    >
```

Comment:
spanning across multiple lines is unnecessary here.

```suggestion
                    <Alert severity="warning">
```